android-app/app/src/main/java/de/dhbw/ka/se/fibo/CreateAccountActivity.java

```diff
@@ -55,9 +103,89 @@ private void initializeButton() {
     }
 
     private void registerButtonClicked() {
-        Intent i = new Intent(CreateAccountActivity.this,
+        i = new Intent(CreateAccountActivity.this,
                 MainActivity.class);
-        startActivity(i);
-        finish();
+        create_account_email = findViewById(R.id.create_account_email);
+        String email = create_account_email.getText().toString();
+        passwordField = findViewById(R.id.create_account_password_layer);
+        emailField = findViewById(R.id.create_account_email_layer);
+        create_account_password = findViewById(R.id.create_account_password);
+        String password = create_account_password.getText().toString();
```

Comment:
```suggestion
        String password = Objects.requireNonNull(binding.createAccountPassword.getText()).toString();
```

please continue to use bindings instead of findViewById (including above the marked line) and wrap the getText() value with requireNonNull